queue/test/fiskaltrust.Middleware.Localization.QueueDE.IntegrationTest/fiskaltrust.Middleware.Localization.QueueDE.IntegrationTest.csproj

```diff
@@ -5,14 +5,6 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
```

Comment:
Are there any WCF tests in here? If yes, I guess we need to keep these things :)

Reply:
in the `middleware` repo neither of these constants are used as far as I could tell.